cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/case.yaml

```diff
@@ -2,9 +2,9 @@ name: ibm-spectrum-scale-csi-operator
 specVersion: 1.0.0
 version: 2.10.0
 appVersion: 2.10.0
-description: "Represents a deployment of the IBM CSI Spectrum Scale driver."
-displayName: "IBM CSI Spectrum Scale Driver CASE" 
-displayDescription: "Represents a deployment of the IBM CSI Spectrum Scale driver."
+description: "Represents a deployment of the IBM CSI Storage Scale driver."
```

Comment:
@Jainbrt please check this part if this works for case

Reply:
changed in the last commit.

Reply:
@deeghuge as discussed we can raise a new PR for case bundle.